sdcm/cluster.py

```diff
@@ -556,9 +556,9 @@ def download_prometheus_data_dir(self):
         self.remoter.receive_files(src=self.prometheus_data_dir, dst=dst)
 
     def _write_prometheus_cfg(self, targets):
-        targets_list = ['%s:9103' % str(ip) for ip in targets]
-        scylla_targets_list = ['%s:9100' % str(ip) for ip in targets]
-        node_exporter_targets_list = ['%s:9180' % str(ip) for ip in targets]
+        scylla_targets_list = ['%s:9100' % str(ip) for ip in targets['db_nodes'] + targets['loaders']]
+        node_exporter_targets_list = ['%s:9180' % str(ip) for ip in targets['db_nodes'] + targets['loaders']]
+        loader_targets_list = ['%s:9103' % str(ip) for ip in targets['loaders']]
```

Comment:
The variables' name and ports don't match.

- node_exporter: 9100
- prometheus API: 9180
- collectd_exporter: 9103

collectd_exporter was used by old scylla, we won't support that?

Reply:
on loaders only - for stress metrics

Reply:
No need for collectd on scylla nodes anymore.
Also, prometheus ignore the orig-scylla job.

Reply:
Got it.

I already approved the PR.